docs/api/structures/process-metric.md

```diff
@@ -11,6 +11,8 @@
   * `Pepper Plugin`
   * `Pepper Plugin Broker`
   * `Unknown`
+* `serviceName` String (optional) - Name of the service hosted in the utility process.
+    Examples: `Audio Service`, `Content Decryption Module Service`, `Network Service`, `Video Capture`, etc.
```

Comment:
Do we have a whole list of services? Can we use an `enum` here instead of a `string`?

Reply:
I was trying to get it, but it's complicated, there a lot of services and I don't know, which ones apply to Electron

Reply:
If there is interest only for the services and not all the utility process then `content::ChildProcessData::sandbox_type` can be used which is a predefined enum for each service.

Reply:
for the pepper plugin host this given the plugin name, so it cannot be an enum

Reply:
well pepper plugin host is not a service but a generic child process, I was suggesting the sandboxed enum because of 

> serviceName` String (optional) - Name of the service hosted in the utility process.

but I see that you have corrected it to `name String (optional) - The name of the process. i.e. for plugins it might be Flash.`. Thanks this looks better.